presto-spi/src/main/java/com/facebook/presto/spi/StandardErrorCode.java

```diff
@@ -65,6 +65,7 @@
     INVALID_ANALYZE_PROPERTY(0x0000_002A, USER_ERROR),
     GENERATED_BYTECODE_TOO_LARGE(0x0000_002B, USER_ERROR),
     WARNING_AS_ERROR(0x0000_002C, USER_ERROR),
+    INVALID_ARGUMENTS(0x0000_002D, USER_ERROR),
```

Comment:
Hmm we don't have https://github.com/martint/trino/commit/1f40322942c92c50a410c65f42f36f0629d2d4fb so it's questionable whether this makes sense in isolation or if we should use some other existing error code.

Martin's reasoning seems sound. @arhimondr  @rongrong  @kaikalur do you have any opinions on direction for error codes?